lib/active_admin_import/model.rb

```diff
@@ -5,7 +5,7 @@ class Model
     include ActiveModel::Validations
     include ActiveModel::Validations::Callbacks
 
-    validates :file, presence: {message: Proc.new { I18n.t('active_admin_import.no_file_error') }},
+    validates :file, presence: { message: Proc.new { I18n.t('active_admin_import.no_file_error') }},
```

Comment:
Line is too long. [100/80]<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.